spec/controllers/idv/phone_controller_spec.rb

```diff
@@ -149,6 +149,7 @@
         put :create, params: { idv_phone_form: { phone: international_phone } }
 
         expect(flash[:warning]).to be_nil
+        expect(flash[:error]).not_to be_nil
```

Comment:
Couple minor comments:

- The `flash[:warning]` assertion might be left over from some [old code](https://github.com/18F/identity-idp/pull/1048/files#diff-be7ca831f3bdf55f9126f80c863f97c94676b5fcb0590f249ca7d66765712fd3L52)? In any case, it wouldn't be set by default anyways, so I'm not sure there's much value in keeping it.
- We could assert the actual text of the message we're expecting, since there's a bit of logic that has to go right in order for it to be the error message we want.

```suggestion
        expect(flash[:error]).to eq t('errors.messages.must_have_us_country_code')
```

We can also probably repeat this for the test case above. This test case is checking for a U.S. number, while the one starting on line 141 is checking for a valid number, where we could assert:

```rb
expect(flash[:error]).to eq t('errors.messages.improbable_phone')
```